stlab/channel.hpp

```diff
@@ -29,8 +32,9 @@ template <typename> class receiver;
 
 enum class process_state {
     await,
-    await_try,
+    hold,
```

Comment:
What is hold and closed? await_try was await if a value is available, otherwise yield (allowing for an interruptible task).